TABTestKit/Classes/Contexts/InteractionContext.swift

```diff
@@ -35,11 +35,19 @@ public extension InteractionContext {
 	}
 	
 	func state(of element: Element, is states: ElementAttributes.State...) {
-		states.forEach { element.await($0) }
+    #if swift(>=5.5)
+      states.forEach { element.waitFor($0) }
+    #else
+		  states.forEach { element.await($0) }
```

Comment:
Has the indentation gone a bit mad on this one and the one below?

Reply:
Yeah I need to fix the indentation. I'll do that soon